.golangci.yml

```diff
@@ -8,3 +8,6 @@ run:
     - internal/connector/internal/api/admin/private
   # timeout for analysis, e.g. 30s, 5m, default is 1m
   timeout: 10m
+linters:
+  enable:
+    - bodyclose
```

Comment:
Is it possible to add more linting rules e.g on using loop variables without pinning them? Or that's not possible?

Reply:
I can give a look to see if there is any other option, but ATM I see only 2 options:

1. put here `enable-all: true` to enable all the linters
2. list all the linters you want to enable under `enable` as in this PR

Reply:
Thanks, let me know if you find something. 

I'd say option 2 is better since we are a bit in control of want we want to check and we'll be progressively enabling some lints as we go. 
Option 1 might be too hard to make happen due to the massive errors we'll see.

For a start, I was only thinking of enabling the linting so that we fail if someone tries to re-use a loop variable in a way that's likely to cause issues.

Reply:
Thanks.